.github/workflows/on-push.yml

```
@@ -160,11 +160,36 @@ jobs:
       run: cd apps/e2e && npx playwright install
     - name: Run Playwright tests
       run: cd apps/e2e && rushx test:ci
+    - name: Install lftp
+      if: always()
+      run: |
+        [ -d apps/e2e/playwright-report ] || exit 0
+        sudo apt-get update
+        sudo apt-get install lftp
```

```suggestion
      run: |
        added
        [ -d apps/e2e/playwright-report ] || exit 0
        sudo apt-get install edited
```

Just testing suggestions